lib/helpers/utils.js

```diff
@@ -132,6 +138,18 @@ export function safeSpawnSync(command, args, options) {
     if (DEBUG_MODE) {
       console.log(`cdxgen lacks execute permission for ${command}`);
     }
+    // Print language specific commands suggestion
+    const normalizedCommand = basename(command.trim());
```

Comment:
This style of normalization before checking would lead to security bugs, since the adversary can control the path to an executable.